### DISCUSSION TITLE
Installing useNProgress in Nuxt 3

### DISCUSSION THREAD

**Discussion by janfrl** [vueuse/vueuse#1981] [Q&A] [answered]
Hi,

I want to use useNProgress in a Nuxt 3 project, but get the error 'Could not resolve "nprogress"' when starting the dev server while vs code doesn't throw any errors. What am I missing?

I installed the following packages:
@vueuse/nuxt
@vueuse/integrations

and used it like in the [demo](https://vueuse.org/integrations/usenprogress/).

Reproduction: https://stackblitz.com/edit/nuxt-starter-1nlcun?file=package.json

Not sure if this is related to https://v3.nuxtjs.org/guide/going-further/esm/#troubleshooting-esm-issues

**Comment by janfrl** [comment]
I tried installing nprogress like suggested in https://github.com/vueuse/vueuse/issues/1306 but still no luck.

**Comment by webfansplz** [comment, marked as answer]
You should install the `nprogress` first:

```json
 "dependencies": {
    "nprogress": "^0.2.0",
  }
```

The function doesn't has SSR support now :

You can do this at local: https://stackblitz.com/edit/nuxt-starter-kejkyh?file=useProgress.ts,package.json,app.vue

Or waiting the PR to merged : #1990

**Reply by janfrl** [reply]
Thanks, that works :)
